packages/devextreme/js/ui/dialog.js

```diff
@@ -174,6 +154,26 @@ export const custom = function(options) {
         }
     }, options.popupOptions));
 
+    each(options.buttons || [DEFAULT_BUTTON], function() {
+        const action = new Action(this.onClick, {
+            context: popupInstance
+        });
+
+        popupToolbarItems.push({
+            toolbar: 'bottom',
+            location: devices.current().android ? 'after' : 'center',
+            widget: 'dxButton',
+            options: extend({}, this, {
+                onClick: function() {
+                    const result = action.execute(...arguments);
+                    hide(result);
+                }
+            })
+        });
+    });
+
+    popupInstance.option('toolbarItems', popupToolbarItems);
```

Comment:
why it's necessary to pass option on runtime instead of setting it on init as before

Reply:
note: @Zedwag and I looked at the ways of doing it and decided to leave it as is